Update use case (#228)
AY2425S1-CS2103T-F14a-2/tp: docs/DeveloperGuide.md

Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`

| Priority | As a …​                                     | I want to …​                                                 | So that I can…​                                                       |
|----------|---------------------------------------------|--------------------------------------------------------------|-----------------------------------------------------------------------|
| `* * *`  | user                                        | add customers' details such as name, contact number, address | remember key and essential details of customers for future deliveries |
| `* * *`  | user                                        | add orders related to a customer                             | keep track of deliveries to be made                                   |
| `* * *`  | user                                        | add details to orders, such as delivery date and item        | know what needs to be delivered and when                              |

> | `* * *`  | user                                        | mark orders as completed                                     | track which orders are still outstanding                              |

Reviewer: isn't this kx part?

Author: oh

Author: deleted

1.  User wants to delete an order.
2.  User chooses to <ins>view all orders of a specified customer from contacts list (UC04)</ins>.
3.  User deletes the order by the customer name and the order index.
4.  SellSavvy updates the status of the order and indicates that the action is successful.

Use case ends.

> **Use case 8: Filter order list by order status**

Reviewer: ![image](https://github.com/user-attachments/assets/80bffcfc-55fe-4306-b7f9-ca8ab426a81e)

i not sure but can we move it above? to below listorder then those below we can change to chooses to view all orders or filtered orders. can someone help on this? :(

Reviewer: Nvm u can ignore this first not ipt

* **System**: SellSavvy
* **Actor**: User
* **Use Case**: UC08 - Filter an Order List by Order Status
* **Guarantees**:
    * Orders with specified status made by specific customer will be displayed as a list, if input parameters are valid.

**MSS**

1.  User wants to filter an order list.
2.  User chooses to <ins>view all orders of a specified customer from contacts list (UC04)</ins>.
3.  User filters the order list of the customer by the status keyword.

> 4.  SellSavvy displays the orders with specified status in a list in GUI.

Reviewer: i not sure but 'with the specified status'?

* **Mainstream OS**: Windows, Linux, Unix, MacOS
* **Order**: Agreement made by customers with user on delivery of product
* **Private contact detail**: A contact detail that is not meant to be shared with others

> * **Status**: The current fulfilment condition of an order, such as completed or pending.

Reviewer 2: ```suggestion
* **Status**: The current fulfilment condition of the delivery of an order, namely completed or pending.
```

Reviewer 2: Negligible, although Im not sure if "condition" is the right word, but not much difference I guess.
